gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -0,0 +1,25 @@
+openapi: "3.0.2"
```

```suggestion
openapi: "3.1.0"
```

We can use the latest version of OpenAPI released in 2021